src/common/filesystembase.cpp

```diff
@@ -480,6 +480,11 @@ bool FileSystem::isLnkFile(const QString &filename)
     return filename.endsWith(QLatin1String(".lnk"));
 }
 
+bool FileSystem::isExcludeFile(const QString &filename)
+{
+    return filename.endsWith(QStringLiteral("sync-exclude.lst")) || filename.endsWith(QStringLiteral("exclude.lst"));
```

Comment:
I guess I do not understand why `endsWith` instead of an equal comparison ?
we are going to accept names we may not want

Reply:
> I guess I do not understand why `endsWith` instead of an equal comparison ?
> we are going to accept names we may not want

@mgallien My plan was to also catch nested `sync-exclude.lst`, but, I will remove it. If someone reopens this issue later with nested `sync-exclude.lst` causing problems, I will then just bring `endsWith` back.

Reply:
In that case, you can do the comparison such that you are sure not to catch filenames like `test.exclude.lst` but accept `test/.exclude.lst` or `.exclude.lst`

Reply:
or if I did not understood correctly accept:

- `.sync-eclude.lst`
- `test/.sync-exclude.lst`

but refuse
- `test.sync-exclude.lst`
- `test/rover.sync-exclude.lst`

Reply:
@mgallien makes sense, will do that